package.json

```diff
@@ -292,7 +292,9 @@
     "scripts": {
         "vscode:prepublish": "npm run compile",
```

Comment:
Should we change this to `npm run recompile` too?

Reply:
Automated builds _should_ be working from a clean repo, but this sounds like a good idea. 
"Only **YOU** can prevent publishing garbage!"

Reply:
I run  `npm run package` before pushing, this will help ease some of that pain for me :)